ime/app/src/main/res/values/settings_keys_dont_translate.xml

```diff
@@ -354,6 +354,7 @@
     <string name="settings_key_persistent_layout_per_package_id_mapping">settings_key_persistent_layout_per_package_id_mapping</string>
 
     <string name="settings_key_quick_fix">quick_fix</string>
+    <string name="settings_key_quick_fix_second_disabled">quick_fix_second_disabled</string>
```

Comment:
settings keys should start with `settings_key_`. And in general, are the same string as the resource name. So: `<string name="settings_key_quick_fix_second_disabled">settings_key_quick_fix_second_disabled</string>`

Reply:
Done! I actually used `settings_key_quick_fix` as reference, and if you notice, its string is wrong as well. Should we change that, too?